### PR TITLE
Reward pot

### PR DESCRIPTION
**Pull request by jonnyblankinship**
Added resetPot function in the Pot class and added Pot value to winner in showWinner function.

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -250,6 +250,9 @@ def rewardPot(self, player):
     player.addMoney(self.money)
     self.money = 0
 
+  def resetPot(self):
```
Comment (bennettapps) [inline source/CardGames.py:253]: doesn't the rewardPot function already reset the pot?

File: test/test_rewardPot.py
Comment (bennettapps) [inline test/test_rewardPot.py:1]: i don't see any assertions in your test?

**Review by dhale27 — APPROVED** [review]
Originally I thought rewarding the winner should go in the Play() function or it's own but putting this in the showWinner() function is smart since it returns the whole player object.

File: source/CardGames.py (hunk 1 above)
Comment (jonnyblankinship) [reply source/CardGames.py:253]: Yes, but there are situations where we may want to reset the pot without actually rewarding the money to anyone, such as if the game was ended prematurely.

File: test/test_rewardPot.py
Comment (jonnyblankinship) [reply test/test_rewardPot.py:1]: I didn't use assertions, I just screenshot the output which proved functionality.

**Review by bennettapps — APPROVED** [review]

**Pull request merged**